Doc/whatsnew/3.9.rst

```diff
@@ -1289,6 +1289,10 @@ New Features
   representation of a function-like object.
   (Patch by Jeroen Demeyer in :issue:`37645`.)
 
+* Added :c:func:`_PyObject_CallOneArg` for calling an object with one
```

Comment:
Oh wait, you mean the public https://docs.python.org/3.9/c-api/call.html#c.PyObject_CallOneArg ?

You're right, this one is part of the public C API. I wrote a compatibility implementation for Python 3.8 and older in https://github.com/pythoncapi/pythoncapi_compat ;-)

Reply:
@vstinner Oops. sorry it was PyObject_CallOneArg  ;) I updated the PR